python/llm/example/CPU/HF-Transformers-AutoModels/Model/minicpm/README.md

```diff
@@ -0,0 +1,70 @@
+# minicpm
```

Comment:
All minicpm -> MiniCPM

Besides, do not forgot to update the table [here](https://github.com/intel-analytics/ipex-llm/blob/main/README.md#verified-models) and [here](https://github.com/intel-analytics/ipex-llm/blob/main/docs/readthedocs/source/index.rst#verified-models)